wire-ios-sync-engine/Source/Data Model/Conversation+Deletion.swift

```diff
@@ -62,23 +62,26 @@ extension ZMConversation {
 
             if response.httpStatus == 200 {
 
+                var conversation: ZMConversation?
                 contextProvider.syncContext.performGroupedBlock {
+                    conversation = ZMConversation.fetch(
+                        with: conversationId,
+                        domain: nil,
+                        in: contextProvider.syncContext
+                    )
+                }
 
-                    guard let conversation = ZMConversation.fetch(
-                            with: conversationId,
-                            domain: nil,
-                            in: contextProvider.syncContext
-                    ) else {
-                        return
-                    }
+                guard let conversation else {
+                    return completion(.success(()))
```

Comment:
@typfel is it correct to call completion with success in this case?

Reply:
So in this case the conversation we wanted to deleted doesn't exist locally when we process the response. Yeah, I I suppose we can consider that a success.